htdocs/comm/propal/list.php

```diff
@@ -1845,7 +1845,7 @@
 			if (!$i) {
 				$totalarray['pos'][$totalarray['nbfield']] = 'p.total_ht';
 			}
-			$totalarray['val']['p.total_ht'] += $obj->total_ht;
+			isset($totalarray['val']['p.total_ht'])?$totalarray['val']['p.total_ht'] += $obj->total_ht:$totalarray['val']['p.total_ht'] = $obj->total_ht;
```

Comment:
![image](https://user-images.githubusercontent.com/3624836/191590797-e3e7e30b-3149-4db5-8ea8-82692ccae6bd.png)
$totalarray['val']['p.total_ht'] is set here, it's not enough ?

Reply:
in my instance i have a php warning
![image](https://user-images.githubusercontent.com/34568357/191591759-691326e3-323e-405e-9a30-774cd071577a.png)

Reply:
maybe an external module resetting totalarray in hook printFieldListTitle ?

Reply:
in doubt i have comment lines 
`$reshook = $hookmanager->executeHooks('printFieldListTitle', $parameters); // Note that $action and $object may have been modified by hook
	print $hookmanager->resPrint;`

the result is the same

Reply:
can you find the moment where totalarray is unset using xdebug step by step ?